runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -754,6 +754,28 @@ void taskLookup(ChatMessage chatMessage, String message)
 			return;
 		}
 
+		if (task == null)
```

Comment:
I don't think this can happen

Reply:
It can. See stack trace https://github.com/runelite/runelite/issues/8047#issuecomment-469944438. It's when the HTTP API returns back an empty string for the task, and trying to deserialize "" into a task object.

Reply:
To repro:
1. have an account without slayer task
2. check enchanted gem so the RuneLite HTTP API knows you have no slayer task
3. perform `!task`